packages/backend/src/open_payments/payment/incoming/routes.test.ts

```diff
@@ -80,6 +80,7 @@ describe('Incoming Payment Routes', (): void => {
   })
 
   describe('get/list', (): void => {
+    console.log('incoming get/list walletAddress=', walletAddress)
```

Comment:
Is there a reason you want to keep this?

Reply:
No, for all the console.logs introduced in this PR I intended to remove them 🤦‍♂️